File: UnityClient/Assets/src/lib/action/CaptureUnitAction.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using Assets.src.lib.entities;


namespace Assets.src.lib.action
{
    public class CaptureUnitAction : Action
    {
        public Unit unit { get; set; }
        public Unit enemy { get; set; }
    }
}

File: project/lib/core/action/CaptureUnitAction.js
utils = require("../utils");
Action = require("./Action")


class CaptureUnitAction extends Action{
    constructor(unit, enemy){
        super();
        this.unit = unit;
        this.enemy = enemy;
        this.type = "Capture building";
    }
}


module.exports = CaptureUnitAction;